README-sitedev.md

```diff
@@ -145,3 +145,10 @@ current wiki, but other opinions welcome.
 
 We use bootstrap 3, so far no themes.
 
+### Code quality
+
+HTML code is kept nice looking with [`prettier`](https://prettier.io):
+
+```shell
+npx prettier --write .
```

Comment:
Can we get a bit more detail here? Dont we have to install npx prettier first?

Also, how do you see this getting kept up to date? Any chance for a github action that runs this once in a while? Or whenever and md/html file is changed?

Reply:
I just update the installation docs. There are indeed github actions to automatically run this, but I didn't want to make a first PR too complicated